interface/modules/custom_modules/oe-module-prior-authorizations/composer.json

```diff
@@ -1,27 +0,0 @@
-{
```

Comment:
goal would be to keep this file for:
1. autoloading the psr-4 classes in the module
2. bringing in packages into vendor that are not in the core codebase (in this case, should be none).

Then the package could be built with composer separately and dropped into the directory and should work.